wcivf/apps/elections/templates/elections/elections_view.html

```diff
@@ -27,4 +35,14 @@ <h2>{% trans "Past Elections" %}</h2>
         {% include "feedback/feedback_form.html" %}
     </div>
 
+    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js">
+        var filter_box = document.getElementById('election-filters');
+        var filter_button = filter_box.querySelector(e.target);
+        $('#election-filters > ul > li:nth-child(1) > a').click(function (e) {
```

Comment:
I'm not too sure what this does, or if it is leftover from before and not needed?

Reply:
I was experimenting with preventing page reloads on filter click, but didn't mean to commit. I'll remove. Thanks.